### PR TITLE
Add Dockerfile for alpine

### PR DESCRIPTION
**Pull request by vkill**
Hi there

The Dockerfile has been completed, but the official docker image 500 happened when `require "ngx.re"`, I should wait openresty/docker-openresty/pull/113 merge. 

--- 2019-07-12 updated
The official docker image was fixed pcre bug in [1.15.8.1-3](https://github.com/openresty/docker-openresty/blob/master/CHANGELOG.md#11581-3) , and I was modified the Dockerfile.

### CODE REVIEW

File: alpine/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    unzip \
+    build-base \
+    make \
+    sudo \
+    git \
+    automake \
+    autoconf \
+    libtool \
+    pkgconfig \
+    cmake \
```
Comment (moonming) [inline alpine/Dockerfile]: I think we need `luarocks` too.

Comment (vkill) [reply alpine/Dockerfile]: The luarocks has been built the [alpine-fat](https://github.com/openresty/docker-openresty/blob/master/alpine/Dockerfile.fat) .

Comment (moonming) [reply alpine/Dockerfile]: cool

**Comment by moonming** [comment]
@vkill I am a newbie of alpine, is alpine has package management utility like `yum`? I want to install the package directly instead of compile in docker, what's do you think about that?

**Comment by vkill** [comment]
> is alpine has package management utility like `yum`? 

It's `apk`, you can [search](https://pkgs.alpinelinux.org/packages) package name then `apk add package-name`

**Comment by vkill** [comment]
@moonming I was completed test. please review it. thx.

**Comment by moonming** [comment]
@vkill I think we should `EXPOSE` the ports of APISIX: 9080, 9443

**Comment by vkill** [comment]
Yes, you are right, it's added.

**Comment by moonming** [comment]
merged, thx

**Pull request merged**